### PR TITLE
The Gumshoe detective alt title now starts with his own original suit and coat

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Shadowmech88** [comment]
Could you post an image of it?

**Comment by ghost** [comment]
f438e368-d12d-11e5-9bb3-617aff9c9add 1

**Comment by gbasood** [comment]
100% tested

**Comment by ihadtoregisterforthis** [comment]
500 detective

:shipit:

**Comment by Shadowmech88** [comment]
Is it just me, or does the hat look a little flat?
I feel like maybe it should be a pixel taller.

**Comment by d3athrow** [comment]
why does this remove noir block

**Comment by Shadowmech88** [comment]
@d3athrow 
It looks to me like he simply moved the noir block addition from line 161 to line 143 of security.dm.

File: code/game/jobs/job/security.dm [hunk 1]
```diff
@@ -128,18 +128,27 @@
...
-			H.equip_or_collect(new /obj/item/clothing/suit/storage/det_suit(H), slot_wear_suit)
-			H.equip_or_collect(new /obj/item/clothing/head/det_hat(H), slot_head)
+		H.equip_or_collect(new /obj/item/clothing/under/det(H), slot_w_uniform)
+		if(H.mind.role_alt_title)
+			switch(H.mind.role_alt_title)
+				if("Forensic Technician")
+					H.equip_or_collect(new /obj/item/clothing/suit/storage/forensics/blue(H), slot_wear_suit)
+					H.equip_or_collect(new /obj/item/clothing/shoes/brown(H), slot_shoes)
+				if("Gumshoe")
+					H.mutations += M_NOIR
```
Comment (ghost) [inline code/game/jobs/job/security.dm:143]: Still here, @d3athrow

**Comment by ihadtoregisterforthis** [comment]
@Shadowmech88 Other versions were less appealing
hurr squad

**Comment by NigglyWiggly** [comment]
it should look like

**Comment by ihadtoregisterforthis** [comment]
But must stay far from
edgemaster

**Comment by Shadowmech88** [comment]
The difference between the two is that the detective's fedora has a wider brim. The athiest fedora looks more like a trilby.

**Comment by gbasood** [comment]
con
flicts

**Comment by ihadtoregisterforthis** [comment]
The euphoric fedora is actually a trilby, yes.

**Comment by ghost** [comment]
what but I fixed those

**Pull request closed without merging**